src/test/java/org/apache/sysds/test/functions/federated/paramserv/FederatedParamservTest.java

```diff
@@ -19,23 +19,25 @@
 
 package org.apache.sysds.test.functions.federated.paramserv;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.sysds.common.Types.ExecMode;
+import org.apache.sysds.hops.codegen.SpoofFusedOp;
 import org.apache.sysds.test.AutomatedTestBase;
 import org.apache.sysds.test.TestConfiguration;
 import org.apache.sysds.test.TestUtils;
 import org.apache.sysds.utils.Statistics;
+import org.dmg.pmml.True;
```

Comment:
Why are you introducing this dependency?

Reply:
I made a mistake. I removed it.